Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -85,13 +86,23 @@ void MemoryWidget::CreateWidgets()
   //// Sidebar
 
   // Search
+  m_address_splitter = new QSplitter(Qt::Horizontal);
```

Comment:
`m_address_splitter` doesn't need to be a member variable. It's only used in this function. It can be just a local variable.

Reply:
Good catch!! Fixed.